js/dragquestion.js

```diff
@@ -868,6 +900,9 @@ H5P.DragQuestion = (function ($) {
     self.y = element.y;
     self.width = element.width;
     self.height = element.height;
+    if (self.height < 10) {
```

Comment:
I know the calculation turns out the same, but the code I pointed out establishes a minimum height for elements. It seems like this would be considerably greater calculated in em than in %.